Add tally of fixable problems to `string` and `verbose` formatters (#7539)
stylelint/stylelint: .changeset/selfish-points-notice.md

---
"stylelint": minor
---

> Added: Show a count of fixable problems in the string formatter

Reviewer: [suggest] Can you change the description for wording consistency in CHANGELOG.md?

```suggestion
Added: tally of fixable problems to `string` and `verbose` formatters
```
